db/seeds.rb

```diff
@@ -5,3 +5,42 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+Answer.delete_all
+ApplicationRecord.delete_all
+Keyword.delete_all
+Officer.delete_all
+Question.delete_all
+Rso.delete_all
+
+rso.create!(name: 'Association for Computer Machinery',
+            shortName: 'ACM',
+            category: 'Computer Science')
+
+rso.create!(name: 'Knitting Club',
```

Comment:
For all of the classes you should have a format similar to this: 

Rso.create([
     {name: 'Knitting Club', nickname: 'KNIT'},
     {name: 'Knitting Club1', nickname: 'KNIT1'},
     {name: 'Knitting Club2', nickname: 'KNIT2'},
]);

Don't do them all in separate actions. Also, check the schema.rb to see all of the fields that exist. We're going to need to specify IDs in most of these also none of the categories or RSOs have any keyword data.